src/KristofferStrube.EditorConfigWizard/Shared/HighLightedCode.razor

```diff
@@ -26,8 +30,28 @@
 
     protected override void OnParametersSet()
     {
-        hoverClass = BorderHoverColor is null ? "" : $"{BorderHoverColor.ToString().ToLower()}-hover";
-        colorClass = BorderColor is null ? "" : $"{BorderColor.ToString().ToLower()}";
+        cursor = OnClick is not null ? "pointer" : "default";;
+
+        var hasHover = BorderHoverColor is not null || BorderColor is not null;
+        
+        if (hasHover)
+        {
+            var sb = new StringBuilder();
+            
+            if (BorderHoverColor is not null)
+            {
+                hasHover = true;
+                sb.Append($"{BorderHoverColor.Value.ToString().ToLower()}-hover ");
+            }
+            if (BorderColor is not null)
+            {
+                sb.Append($"{BorderColor.Value.ToString().ToLower()}-hover ");
```

Comment:
This should simply be the color without the "-hover" part.

Reply:
Good catch, thanks. Fixed and rebased.